packages/main/src/themes/NavigationMenuItem.css

```diff
@@ -0,0 +1,54 @@
+
+:host::part(native-li) {
+	padding-left: 0.5rem;
+	width: auto;
+}
+:host::part(native-li)::after {
+	border-radius: 0.375rem;
+	inset: 0;
+ }
+
+ :host {
+	display: flex;
+	align-items: center;
+	width: 100%;
+	box-sizing: border-box;
+	position: relative;
+	height: var(--_ui5_side_navigation_item_height);
+	min-height: var(--_ui5_side_navigation_item_height);
+	border-radius: 0.375rem;
+	transition: var(--_ui5_side_navigation_item_transition);
+	color: var(--sapList_TextColor);
+	font-weight: bold;
+	font-size: var(--sapFontSize);
+	border-bottom: none;
+}
+
+:host a {
```

Comment:
Do we have any other way to distinguish "a" elements or only by tagname?

Reply:
done